aws/account-roles/module.tf

```diff
@@ -110,6 +110,12 @@ variable "delimiter" {
   description = "Delimiter to be used between `namespace`, `environment`, `stage`, `name`"
 }
 
+variable "create_iam_groups" {
```

Comment:
Create an additional variable with a map to define which IAM groups we want to create. Also set the default on this resource to `false`.

Reply:
Will take a look